addons/web_editor/static/tests/banner_tests.js

```diff
@@ -0,0 +1,85 @@
+/** @odoo-module **/
+import { setSelection } from "@web_editor/js/editor/odoo-editor/src/utils/utils";
+import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
+import { patchWithCleanup, nextTick } from "@web/../tests/helpers/utils";
+import { Wysiwyg } from "@web_editor/js/wysiwyg/wysiwyg";
+import {
+    triggerEvent,
+    insertText,
+} from "@web_editor/js/editor/odoo-editor/test/utils";
+
+function onMount() {;
+    const editor = wysiwyg.odooEditor;
+    const editable = editor.editable;
+    editor.testMode = true;
+    return { editor, editable };
+}
+
+let wysiwyg;
+
+QUnit.module(
+    "web_editor",
+    {
+        beforeEach: async function () {
+            const serverData = {
+                models: {
+                    note: {
+                        fields: {
+                            body: {
+                                string: "Editor",
+                                type: "html",
+                            },
+                        },
+                        records: [
+                            {
+                                id: 1,
+                                display_name: "first record",
+                                body: "<p><br></p>",
+                            },
+                        ],
+                    },
+                },
+            };
+            setupViewRegistries();
+            patchWithCleanup(Wysiwyg.prototype, {
+                async init() {
+                    await this._super(...arguments);
+                    wysiwyg = this;
+                },
+            });
+            await makeView({
+                type: "form",
+                serverData,
+                resModel: "note",
+                arch:
+                    "<form>" +
+                    '<field name="body" widget="html" style="height: 100px"/>' +
+                    "</form>",
+                resId: 1,
+            });
+        },
+    },
+    function () {
+
+        QUnit.module("insert a new banner");
+        QUnit.test("should insert a banner with paragraph tag", async function (assert) {
```

Comment:
Didn't you say you meant "followed by a paragraph" rather than "with paragraph"? The distinction matters because "with paragraph" could mean that what matters in your test is that the banner _contains_ a paragraph (which it does), but from what I understand you want to make sure the banner is inserted and _also_ that a paragraph is inserted _after it_.

Reply:
I thought I changed it, but I probably accidentally reverted it while making other changes. Sorry for the inconvenience.